src/views/ModelPlan/TaskList/Payment/ClaimsBasedPayment/index.tsx

```diff
@@ -0,0 +1,577 @@
+                    <Form
+                      className="margin-top-6"
+                      data-testid="payment-claims-based-payment-form"
+                      onSubmit={e => {
+                        handleSubmit(e);
+                      }}
+                    >
+                      <PageHeading
+                        headingLevel="h3"
+                        className="margin-bottom-3"
+                      >
+                        {t('claimSpecificQuestions')}
+                      </PageHeading>
+
+                      <FieldGroup
+                        scrollElement="payClaims"
+                        error={!!flatErrors.payClaims}
+                        className="margin-top-4"
+                      >
+                        <Label htmlFor="payment-pay-claims">
+                          {t('selectClaims')}
+                        </Label>
+                        <p className="text-base margin-bottom-1 margin-top-05">
+                          {t('selectClaimsSubcopy')}
+                        </p>
+                        <FieldErrorMsg>{flatErrors.payClaims}</FieldErrorMsg>
+
+                        <Field
+                          as={MultiSelect}
+                          id="payment-pay-claims"
+                          name="beneficiaries"
+                          options={mappedClaimsBasedPayType}
+                          selectedLabel={t('selectedClaimsOptions')}
+                          onChange={(value: string[] | []) => {
+                            setFieldValue('payClaims', value);
+                          }}
+                          initialValues={initialValues.payClaims}
+                        />
+
+                        {(values?.payClaims || []).includes(
+                          ClaimsBasedPayType.OTHER
+                        ) && (
+                          <FieldGroup
+                            scrollElement="payClaimsOther"
+                            error={!!flatErrors.payClaimsOther}
+                          >
+                            <Label
+                              htmlFor="payment-pay-claims-other"
+                              className="text-normal"
+                            >
+                              {t('selectClaimsOther')}
+                            </Label>
+                            <FieldErrorMsg>
+                              {flatErrors.payClaimsOther}
+                            </FieldErrorMsg>
+                            <Field
+                              as={TextField}
+                              error={flatErrors.payClaimsOther}
+                              id="payment-pay-claims-other"
+                              data-testid="payment-pay-claims-other"
+                              name="payClaimsOther"
+                            />
+                          </FieldGroup>
+                        )}
+
+                        <AddNote id="pay-claims-note" field="payClaimsNotes" />
```

Comment:
The field should be "payClaimsNote".  BE throws error as is not valid field name